python-avd/pyavd/_eos_designs/schema/schema_fragments/bgp_peer_groups.schema.yml

```
@@ -34,6 +34,28 @@ keys:
             documentation_options:
               hide_keys: true
             $ref: "eos_cli_config_gen#/keys/router_bgp/keys/peer_groups/items"
+      mlag_ipv4_vrfs_peer:
+        type: dict
+        description: |-
+          If the names of `mlag_ipv4_vrfs_peer.name` and `mlag_ipv4_underlay_peer.name` are same
+          then all the attributes of key `mlag_ipv4_vrfs_peer` will be ignored.
```

Add something more to the description

```suggestion
        description: |-
          Set this peer group name to use a different peer-group for MLAG peerings in VRFs.
          By default AVD uses the `mlag_ipv4_underlay_peer` peer group for the Underlay and for all the VRFS.
          
          If the names of `mlag_ipv4_vrfs_peer.name` and `mlag_ipv4_underlay_peer.name` are the same
          then all the attributes of key `mlag_ipv4_vrfs_peer` are ignored.
```